src/webui/www/private/setlocation.html

```diff
@@ -68,7 +68,7 @@
 <body>
     <div style="padding: 10px 10px 0px 10px;">
         <p style="font-weight: bold;">QBT_TR(Location)QBT_TR[CONTEXT=TransferListWidget]:</p>
-        <input type="text" id="setLocation" value="" maxlength="100" autocorrect="off" autocapitalize="none" style="width: 370px;" />
+        <input type="text" id="setLocation" value="" maxlength="500" autocorrect="off" autocapitalize="none" style="width: 370px;" />
```

Comment:
how about removing the `maxlength` attribute?

Reply:
Yes, why not, good idea